playbooks/vars/defaults.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-rhos_release: 16.1
```

Comment:
Should we perhaps make a copy of the old `defaults.yaml` to an `example-overrides/local-overrides-osp16-1.yaml` file, if someone ever need to deploy RHOSP16.1?

Reply:
done